pkg/controller/cassandradatacenter/statefulset.go

```diff
@@ -157,6 +191,30 @@ func newSidecarContainer(cdc *cassandraoperatorv1alpha1.CassandraDataCenter, dat
 			{Name: podInfoVolume.Name, MountPath: "/etc/pod-info"},
 		},
 	}
+
+	if len(cdc.Spec.Env) > 0 {
+		container.Env = cdc.Spec.Env
+	}
+
+	if backupSecretVolume != nil {
```

Comment:
I think it'd be preferable to not have the operator deal with provider-specific stuff unless 100% necessary.
This code also seems to imply that the backup destination would always be GCP.

Reply:
As per above, I'll think a more generic location. Clearly that this was done initially for GCP, and other providers will probably not use it....but of course they might.